### PR TITLE
(DOCSP-42007) Add additional option for unsyncable doc limit

### PR DESCRIPTION
**Pull request by krollins-mdb**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-42007

### Staging Links
<br/> - https://deploy-preview-818--app-services.netlify.app/mongodb/internal-database-usage<br/> - https://deploy-preview-818--app-services.netlify.app/sync/data-model/update-schema

Note: the build error is for the known Counters ref that's being weird.

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [x] Did you tag pages appropriately?
  - genre
  - programming_language
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [x] Create a Jira ticket for related docs-realm work, if any

### Release Notes

- **Connect to MongoDB Sources**
  - Internal Database Usage: Add additional option for unsyncable doc limit
- **Device Sync/Define and Update Your Data Model**
  - Update Your Data Model: Add additional option for unsyncable doc limit

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### Animal Wearing a Hat

<br/> - https://deploy-preview-818--app-services.netlify.app/mongodb/internal-database-usage<br/> - https://deploy-preview-818--app-services.netlify.app/sync/data-model/update-schema<br/> - https://deploy-preview-818--app-services.netlify.app/mongodb/internal-database-usage<br/> - https://deploy-preview-818--app-services.netlify.app/sync/data-model/update-schema<br/> - https://deploy-preview-818--app-services.netlify.app/mongodb/internal-database-usage<br/> - https://deploy-preview-818--app-services.netlify.app/sync/data-model/update-schema<br/> - https://deploy-preview-818--app-services.netlify.app/mongodb/internal-database-usage<br/> - https://deploy-preview-818--app-services.netlify.app/sync/data-model/update-schema

### CODE REVIEW

File: source/mongodb/internal-database-usage.txt [hunk 1]
```diff
@@ -70,6 +78,14 @@ App Services stores a information about unsyncable documents in the
    You can read from the ``__realm_sync.unsynced_documents`` collection,
    but you should not modify it in any way.
 
+If Device Sync is paused due to hitting the 100,000 unsyncable document limit,
```
Comment (dacharyc) [inline source/mongodb/internal-database-usage.txt:81]: If we're adding this here, we should probably remove the "You need to manually resolve this by..." at line 70 above.

File: source/mongodb/internal-database-usage.txt [hunk 2]
```diff
@@ -70,6 +78,14 @@ App Services stores a information about unsyncable documents in the
...
+you have two options:
+   
+- :ref:`terminate and re-enable sync <terminating-realm-sync>`, even though
+   the change you're making is a non-breaking change.
```
Comment (dacharyc) [inline source/mongodb/internal-database-usage.txt:85]: We should probably omit the info about breaking changes here, as I don't think it's relevant to the info in this section.

```suggestion
- :ref:`terminate and re-enable sync <terminating-realm-sync>`. 
```

File: source/mongodb/internal-database-usage.txt [hunk 3]
```diff
@@ -70,6 +78,14 @@ App Services stores a information about unsyncable documents in the
    You can read from the ``__realm_sync.unsynced_documents`` collection,
    but you should not modify it in any way.
 
+If Device Sync is paused due to hitting the 100,000 unsyncable document limit,
+you have two options:
+   
+- :ref:`terminate and re-enable sync <terminating-realm-sync>`, even though
+   the change you're making is a non-breaking change. 
+- Contact support and request a temporary increase in the unsyncable
+   documents limit.
```
Comment (dacharyc) [inline source/mongodb/internal-database-usage.txt:87]: It looks like the staged page is doing some weird formatting thing, and I wonder if it's related to the extra space here?
```suggestion
  documents limit.
```

File: source/mongodb/internal-database-usage.txt [hunk 4]
```diff
@@ -4,6 +4,14 @@
 Internal Database Usage
 =======================
 
+.. meta::
+   :description: Learn about how Device Sync interacts with your linked Atlas clusters.
```
Comment (dacharyc) [inline source/mongodb/internal-database-usage.txt:8]: I don't think this description is right for this page. There are some things on this page that don't seem to be Sync-related - the "System Generated User" and "Transactional Locks" sections seem to be unrelated to Sync.

File: source/mongodb/internal-database-usage.txt [hunk 5]
```diff
@@ -4,6 +4,14 @@
...
+   :keywords: Realm
```
Comment (dacharyc) [inline source/mongodb/internal-database-usage.txt:9]: I'm not sure this keyword is right either - nothing on this page seems to be directly related to Realm, and only one of them is related to Device Sync, which is also - not analogous to Realm?

**Review by dacharyc — APPROVED** [review]
This LGTM overall, but there are some tweaks I think we should make before merging, including a formatting thing that is causing one of the pages to render weirdly.

File: source/mongodb/internal-database-usage.txt (hunk 4 above)
Comment (krollins-mdb) [reply source/mongodb/internal-database-usage.txt:8]: Hmm. Yeah, you're right. I need to rethink this.
